locales/base/translation.json

```diff
@@ -2602,13 +2602,16 @@
     "poolCard": {
       "onNetwork": "on {{networkName}}",
       "rate": "Rate (est.)",
+      "yieldRate": "Yield Rate (est.)",
       "reward": "Reward",
       "tvl": "TVL",
       "exitPool": "Exit Pool",
       "addToPool": "Add to Pool",
       "apy": "{{apy}}% APY",
+      "percentage": "{{percentage}}%",
```

Comment:
does this have to be translated?

Reply:
It appears to need a translation based on the [Wiki](https://en.wikipedia.org/wiki/Percent_sign):
>English style guides prescribe writing the percent sign following the number without any space between (e.g. 50%). However, the international System of Units and ISO 31-0 standard prescribe a space between the number and percent sign.

And

> In Turkish and some other Turkic languages, the percent sign precedes rather than follows the number, without an intervening space.